protocol/v2/qbft/controller/future_msg.go

```diff
@@ -17,6 +17,7 @@ func (c *Controller) UponFutureMsg(msg *specqbft.SignedMessage) (*specqbft.Signe
 	}
 	if c.f1SyncTrigger() {
 		c.logger.Debug("triggered f+1 sync",
+			zap.Any("operator-id", c.Share.OperatorID),
```

Comment:
it's not a place for this log. please revert. we can open a issue to add this if needed